packages/ui-extensions/src/types/Button.ts

```diff
@@ -1,14 +1,11 @@
 export interface BaseButtonProps {
-  /**
-   * Destination to link to, renders a Link
-   */
-  to?: string;
   /**
    * The type of button that will be rendered.
    *
    * `primary`: button used for main actions or green-path. Ex: Continue to next step, Discount field
    * `secondary`: button used for secondary actions not blocking user progress. Ex: Download Shop app
    * `plain`: Renders a button that visually looks like a Link
+   *  @default 'primary'
    */
   kind?: 'primary' | 'secondary' | 'plain';
```

Comment:
Does it make sense to choose the word `variant` here? I believe it's used in other frameworks.